portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessDeliveryStructuredContent.testcase

```diff
@@ -65,4 +65,54 @@ definition {
 		}
 	}
 
+	@description = "StructuredContent can be filtered with a set priority"
+	@priority = "5"
+	test StructuredContentContainsUpdatedPriorityValueFilteredByEqualTo {
+		property portal.acceptance = "true";
+
+		task ("Given three web contents are created with default priority") {
+			var ddmStructureId = WebContentStructures.getDdmStructureId(structureName = "content-structure");
+
+			var responseToParse = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title1");
+			var responseToParse = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title2");
+			var responseToParse = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title3");
```

Comment:
all three variables have the same name, how do you differentiate the response after? Do we parse the response of this POST?

Reply:
I think i use different names eg: WC WebContent Title3, WC WebContent Title2...